### DISCUSSION TITLE
Update needed in instructions: Disabling box3 speaker

### DISCUSSION THREAD

**Discussion by Bcukier** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#36] [General]
First, this is awesome.  I've been having a lot of fun customizing.  Thank you!

One set of instructions in "Make it your own" that appears to need updating.  To disable speakers, the instructions say:

![333187044-d72e1bc8-6100-4d8e-ae1c-9cf630c785c4](https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/assets/102839257/ba9d7abe-6eb6-4116-80c2-a1232927a49c)

I don't believe these highlighted lines exist in the current code.
I successfully removed the box speaker by commenting out this line:

`#  media_player: adf_media_player`

**Comment by BigBobbas** [comment]
good spot! thank you , I do need to go through the docs again. Media player wasn't an option when I first started the project, but yes I had totaly forgotten to change that part. I will get that updated and add your solution, thanks for letting me know 👍 
Glad you are enjoying the config, It makes the box so much better that just a voice assistant.
Cheers 
Rob

**Comment by BigBobbas** [comment]
Updated docs 🙏 
https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/blob/main/instructions/make%20it%20your%20own.md#outputting-audio-responses-to-another-media-player-device
